code/modules/mob/living/living_defines.dm

```diff
@@ -1,6 +1,8 @@
 /mob/living
 	see_invisible = SEE_INVISIBLE_LIVING
 
+	appearance_flags = TILE_BOUND|PIXEL_SCALE|KEEP_TOGETHER|LONG_GLIDE

```

Comment:
These are already applied on /atom/movable it'd seem?  https://github.com/VOREStation/VOREStation/blob/5195f715452d7a92f2b911f8217343a856f6ed13/code/game/atoms_movable.dm#L3